src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -726,6 +869,36 @@ def forward(
 
         hidden_states = self.feature_projection(hidden_states)
 
+        if self.config.apply_spec_augment and self.training:
```

Comment:
I'm curious, can we apply spec augment to this model? It's trained on the raw waveform rather than spectrograms, so is this more a conv extracted feature drop out?

Reply:
exactly! spec_augment is applied to the features extracted from the conv layers